### PR TITLE
feat(UI): Added new site role 'CLEARING_EXPERT'

### PR DESCRIPTION
**Pull request by akapti**
[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
 * _Issue #820_ Added a new Role **Clearing Expert**, 
 * fixed bug where in adding **SITE role** to a user was not reflecting in couch db.
 * No changes in dependencies.

Issue: 

### How To Test?
* Adding a new `site role` to a user should reflect in couch db.
* All user with role `Clearing Expert`, should be able to view and edit the `Clearing Request` under their `department`.
* No additional tests implemented.

### Checklist
Must:
- [x] All related issues are referenced in commit messages and in PR

Signed-off-by: akapti <abdul.mannankapti@siemens.com>

### CODE REVIEW

**Comment by mcjaeger** [comment]
(from discussion) there is the point that "team" is different from "admin" or "user" because it denotes a user role. Would there another role name that would suit this better like "clearing expert" or "clearing officer"?

File: frontend/sw360-portlet/src/main/resources/content/Language_vi.properties [hunk 1]
```diff
@@ -1087,7 +1087,7 @@ used.in=Được dùng trong
 used.license=Giấy phép được sử dụng
 user=Người dùng
 user.cannot.edit.a.closed.project=Người dùng không thể chỉnh sửa một dự án đã đóng
-UserGroup=Người dùng: &#10Quản trị viên: &#10Quản trị viên xóa: &#10Quản trị viên ECC: &#10Quản trị viên bảo mật: &#10Quản trị viên SW360
+UserGroup=Người dùng: &#10Quản trị viên: &#10Quản trị viên xóa: &#10Clearing Expert: &#10Quản trị viên ECC: &#10Quản trị viên bảo mật: &#10Quản trị viên SW360
```
Comment (smrutis1) [inline frontend/sw360-portlet/src/main/resources/content/Language_vi.properties:1090]: May be We should add the Vietnamese version of `Clearing Expert:` here.

Comment (nam-np) [reply frontend/sw360-portlet/src/main/resources/content/Language_vi.properties:1090]: Hi, thank you for your notification. 
Could you help me add Vietnamese in this line for "Clearing Expert":
`UserGroup=Người dùng: &#10Quản trị viên: &#10Quản trị viên xóa: &#10Chuyên gia xóa: &#10Quản trị viên ECC: &#10Quản trị viên bảo mật: &#10Quản trị viên SW360`

Tks

Comment (akapti) [reply frontend/sw360-portlet/src/main/resources/content/Language_vi.properties]: @nam-np Thanks for sharing the translation.
@smrutis1 I have added the Vietnamese translation of `Clearing Expert`.

**Pull request merged**